src/pages/components/notification/usage.mdx

```diff
@@ -23,6 +24,23 @@ Notifications are messages that communicate information to the user. The two mai
 
 </AnchorLinks>
 
+## Overview
+
+### When to use
+
+Use notifications to inform users of updates or changes to system status. Communicating with users and providing immediate feedback are important for building trust. While notifications are an effective method of communicating with users, they are disruptive and should be used sparingly.
+
+For more context on when to use each notification type, including modals, refer to the [notifications pattern](/patterns/notification-pattern/). Carbon currently only supports inline, toast, and modal notification types, although some product teams also support banners and notification centers.
```

Comment:
```suggestion
For more context on when to use each notification type, including modals, refer to the [notifications pattern](/patterns/notification-pattern/). Carbon only supports inline, toast, and modal notification types, although some product teams also support banners and notification centers.
```

I feel like we should avoid "currently" here as it'll need to be updated either way when Carbon supports more notification types.